grype/presenter/config.go

```diff
@@ -0,0 +1,44 @@
+package presenter
+
+import "fmt"
+
+// Config is the presenter domain's configuration data structure.
+type Config struct {
```

Comment:
open question: do we want to also include the analysis result data on the config object (matches, image source metadata, etc)? (that would be my preference at a casual glance)

Reply:
we chatted about this offline --we can explore this suggestion later 👍

Reply:
Sweet 👍